### ISSUE TITLE
Bug when reading .fem file containing CBUSH element

### ISSUE THREAD

**Issue by Clemoreau** [nschloe/meshio#1113]
Hello everyone,

When I read a .fem file generated by Hypermesh (nastran format), it gives the following error when it comes to reading a CBUSH element (that I don't need because I only want to import mesh):
```
Traceback (most recent call last):
  File "c:/xxxxxxxxx/test_meshio.py", line 6, in <module>
    mesh = meshio.read(path_fem_nok)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python38\lib\site-packages\meshio\_helpers.py", line 69, in read
    return reader_map[file_format](filename)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python38\lib\site-packages\meshio\nastran\_nastran.py", line 49, in read
    out = read_buffer(f)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python38\lib\site-packages\meshio\nastran\_nastran.py", line 177, in read_buffer
    add_cell(keyword, cell, cell_type, cell_ref)
  File "C:\Users\PC\AppData\Local\Programs\Python\Python38\lib\site-packages\meshio\nastran\_nastran.py", line 80, in add_cell
    assert cell_type in ["tetra", "pyramid", "wedge", "hexahedron"]
AssertionError
```
It works fine when I manually remove the lines related to the "CBUSH" element (see after) and read again the file. However I would like not to do it for all the .fem files, plus I also need this data because these .fem files can also launch FEA calculation on Hypermesh.
```
$$  CBUSH Elements
$$
CBUSH    1328173     174 1542965 1542962                              13
CBUSH    1328174     173 1542964 1542963                              14
$
```
Is it somehow possible to ignore any content that is not related to mesh definition ?

Thank you very much for your help,

Clément

**Comment by nschloe** [comment]
@michejansen Can you have a look?

**Comment by michejansen** [comment]
Sure, that seems to be the error that should have been fixed by #1097, but this fix is only available in the latest release (4.4.5) if I'm not mistaken. @Clemoreau Can you check if you still have this problem with the latest release of meshio?

**Comment by Clemoreau** [comment]
Indeed, I used to work with the 4.4.3 version. Now it works with the 4.4.5. 
Thank you for your prompt response !
